lib/array_equals.rb

```diff
@@ -1,5 +1,9 @@
 # Determines if the two input arrays have the same count of elements
 # and the same integer values in the same exact order
 def array_equals(array1, array2)
```

Comment:
The assignment asks you not to use `array1 == array2` but rather write the code that `==` is doing.